src/components/workEntry.js

```diff
@@ -1,24 +1,23 @@
 import React from 'react';
 import '../css/workEntry.css'
 
-function workHistory(props) {
-  const { company, title, image, started, ended, description } = props
-  const startDate = new Date(started)
-  const endDate = new Date(ended)
+function workEntry({entry}) {
+  const startDate = new Date(entry.started)
```

Comment:
Since we're using an object now, we need to define *where* `started` comes from. It's an item in `entry`.